base/common/src/test/java/com/netscape/certsrv/system/SecurityDomainSubsystemTest.java

```diff
@@ -0,0 +1,50 @@
+package com.netscape.certsrv.system;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class SecurityDomainSubsystemTest {
+
+    private static SecurityDomainSubsystem before = new SecurityDomainSubsystem();
+    private static SecurityDomainHost host = new SecurityDomainHost();
+
+    @Before
```

Comment:
The `SecurityDomainHost` object are stored by their IDs in the `SecurityDomainSubsystem` object so there's no duplicate here, but maybe for clarity/consistency we should use `@BeforeClass` in all tests?

Reply:
I'm planning a follow up change where I tidy up the tests a bit. I'm going to replace the `Asserts.assert$SOMETHING()` with static imports, for two reasons. First, I think static imports of assertions in tests makes the code less cluttered and easier to follow, and second when/if the time comes to do #3586 the assertions have moved, to another package. With a static import only the import needs to change, the actual code remains unchanged, which should reduce the size of the diff for that change considerably.

Also, I found one bit of code that makes the JSON pretty print so I'm going to do that for all the classes, currently it prints horribly.

Anyway, yes, I'll do `@BeforeClass` conversion in a follow up change. :-)